apps/nextjs-app/package.json

```diff
@@ -75,6 +75,7 @@
     "cross-env": "7.0.3",
     "dotenv-flow": "3.2.0",
     "dotenv-flow-cli": "1.0.0",
+    "edit-json-file": "1.7.0",
```

Comment:
Whu these are in nextjs-app? Perhaps They could belong to the package.json in the root folder? because looks like only the publishing script is using it and it's unrelated to any app. Anyway the changesets could be the correct way to publish from monorepo... I copied the publish scripts from hobbies just to be able to release quickly.